README.md

```
@@ -1,55 +1,58 @@
-```
+   ```bash
+   npx skills add https://github.com/agentworkforce/skills --skill orchestrating-agent-relay
+   ```
 
-3. Tell your agent to use it
+3. Tell your agent to use it:
 
-```
-use the orchestrating-agent-relay skill to spawn a claude and codex agent and [YOUR_TASK]
-```
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)